package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "make clean",
     "help": "make help",
     "refresh": "make refresh",
-    "test": "make test"
+    "test": "mocha -R list"
```

Comment:
Please don't change the reported used by `npm test` and revert this line. If it helps you to use a different reporter during development, you can always invoke mocha manually instead of calling `npm test`.